frame/acronym.tex

```diff
@@ -13,21 +13,35 @@
 % Acronyms that are used quite commonly
 % If they are not used, they are ignored in the glossary
 \acr{API}{Application Programming Interface}
+\acr{AD}{Active Directory}
 \acr{BL}{Business Logic}
+\acr{BPMN}{Business Process Model and Notation}
 \acr{CLI}{Command Line Interface}
 \acr{DOM}{Document Object Model}
+
```

Comment:
Warum die ganzen Leerzeilen in dieser Datei?

Reply:
aufgrund des mergekonfliktes, hatte ich nicht sauber aufgelöst, danke.